atomic_reactor/util.py

```diff
@@ -953,3 +953,24 @@ class LabelFormatter(string.Formatter):
     """
     def get_field(self, field_name, args, kwargs):
         return (self.get_value(field_name, args, kwargs), field_name)
+
+
+def do_retries(method, url, client_statuses=HTTP_CLIENT_STATUS_RETRY,
```

Comment:
it can return session

I don't think you will be able to use 'raise_for_status' as you don't have request anymore but session, which raises by itself already